### PR TITLE
add NODERS rpc,api,grpc,jsonrpc

### PR DESCRIPTION
**Pull request by aleksandrnoders**

### CODE REVIEW

File: lambda/chain.json [hunk 1]
```diff
@@ -96,12 +104,20 @@
...
       }
     ],
     "evm-http-jsonrpc": [
       {
         "address": "https://jsonrpc-lambda.d-stake.xyz",
         "provider": "D-stake"
+      },
+      {
+        "address": "https://lambda-jsonrpc.noders.services",
+        "provider": [NODERS]TEAM"
```
Comment (reversesigh) [inline lambda/chain.json:120]: Missing starting comma for provider name here

Comment (aleksandrnoders) [reply lambda/chain.json]: Corrected

File: stratos/chain.json [hunk 2]
```diff
@@ -112,6 +120,9 @@
       {
         "address": "stratos.grpc.nodersteam.com:9090",
         "provider": "[NODERS]TEAM"
+      {
```
Comment (reversesigh) [inline stratos/chain.json:123]: Close the set above this. Are you also running two grpc end points or was this intended to be a replacement?

Comment (aleksandrnoders) [reply stratos/chain.json:123]: also fixed :), these are two different nodes, maybe we will remove the one above later

File: stratos/chain.json [hunk 3]
```diff
@@ -88,11 +88,11 @@
       {
         "address": "https://rpc-stratos.whispernode.com:443",
         "provider": "WhisperNode 🤐"
+      },
       {
         "address": "https://stratos-rpc.noders.services:443",
         "provider": "[NODERS]TEAM"
       }
```
Comment (reversesigh) [inline stratos/chain.json:95]: @JeremyParish69 I'm a little stumped, this change looks correct but when I view the file it's still showing me the old commit.

**Pull request merged**